tests/e2e/main_test.go

```diff
@@ -77,3 +82,49 @@ func TestLintMetrics(t *testing.T) {
 		t.Fatalf("the problems encountered in Lint are: %v", problems)
 	}
 }
+
+func TestDefaultCollectorMetricsAvailable(t *testing.T) {
```

Comment:
Actually I don't have local environment.

But AFASK, if it works in CI, then it should work in local as well?

Reply:
Does not work for me locally, fails to find resources.

cc @tariq1890 do you mind running it locally as well, thanks!

Reply:
@LiliC The code has been updated to check the scan error.

Maybe it could give some clue why the test failed in local environment :)

Reply:
@lilic When testing locally, rebased on my PR #1065 (because without it I'm not able to run e2e locally), these tests pass:

```
start e2e test for kube-state-metrics
=== RUN   TestIsHealthz
--- PASS: TestIsHealthz (0.00s)
=== RUN   TestLintMetrics
--- PASS: TestLintMetrics (0.01s)
=== RUN   TestDefaultCollectorMetricsAvailable
--- PASS: TestDefaultCollectorMetricsAvailable (0.01s)
PASS
ok  	k8s.io/kube-state-metrics/tests/e2e	0.031s
```
I can provide you with the full log if needed.